lib/src/emoji_picker.dart

```diff
@@ -151,10 +156,12 @@ class _EmojiPickerState extends State<EmojiPicker> {
       if (widget.config.showRecentsTab) {
         _addEmojiToRecentlyUsed(emoji).then((value) {
           if (category != Category.RECENT) {
-            setState(() {
-              // rebuild to update recent emoji tab
-              // when it is not current tab
-            });
+            if (mounted) {
+              setState(() {
+                // rebuild to update recent emoji tab
+                // when it is not current tab
+              });
+            }
```

Comment:
I don't think we need to check here if the widget is already mounted or not, because if the widget is already disposed I doubt that any emoji can get selected. What do you think?

Reply:
Maybe we should just to be safe.

The user might pop the page in `onEmojiSelected` callback, then when `_addEmojiToRecentlyUsed` is resolved the page might be already disposed.

Reply:
Okay it can't hurt, thank you 🙇